sync.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# x86_64 or arm64
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="-arm64"
+
+# security or no security
+[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="-no-secty"
+
+# redis or mongo
+[ "$FOR_REDIS" = true ] && USE_REDIS="-redis"
+
+# nightly or other release
+USE_RELEASE=${RELEASE:-nightly-build}
+if [ "$USE_RELEASE" = "nightly-build" ]; then
+     COMPOSE_FILE="docker-compose-nexus${USE_REDIS}${USE_NO_SECURITY}${USE_ARM64}.yml"
+else
+     COMPOSE_FILE="docker-compose${USE_REDIS}-${USE_RELEASE}${USE_NO_SECURITY}${USE_ARM64}.yml"
+fi
+
+wget -q "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/${USE_RELEASE}/compose-files/${COMPOSE_FILE}"
```

Comment:
On my system, without the `-O "${COMPOSE_FILE}"` option, when I re-run `deploy-edgeX.sh`  I get multiple dot versions of the compose file download to my system, which creates problems later.

Reply:
added